x-pack/plugins/apm/server/routes/errors/erroneous_transactions/get_top_erroneous_transactions.ts

```diff
@@ -112,17 +119,20 @@ async function getTopErroneousTransactions({
 
   return (
     res.aggregations?.top_five_transactions.buckets.map(
-      ({ key, doc_count: docCount, sample, timeseries }) => ({
-        transactionName: key as string,
-        transactionType: sample.hits.hits[0]._source.transaction?.type,
-        occurrences: docCount,
-        timeseries: timeseries.buckets.map((timeseriesBucket) => {
-          return {
-            x: timeseriesBucket.key + offsetInMs,
-            y: timeseriesBucket.doc_count,
-          };
-        }),
-      })
+      ({ key, doc_count: docCount, sample, timeseries }) => {
+        const source = sample.hits.hits[0]._source as APMError;
```

Comment:
Not required, but I don't think you need to manually type the source, do you?

```suggestion
        const source = sample.hits.hits[0]._source;
```

Reply:
I needed to type it, if not source is type unknown